org.jacoco.core.test/src/org/jacoco/core/test/validation/ExceptionsTest.java

```diff
@@ -90,6 +94,7 @@ public void testCoverageResult() {
 		assertLine("implicitExceptionFinally.before", ICounter.FULLY_COVERED);
 		assertLine("implicitExceptionFinally.exception", ICounter.NOT_COVERED);
 		assertLine("implicitExceptionFinally.after", ICounter.NOT_COVERED);
+		assertLine("implicitExceptionFinally.finally", ICounter.EMPTY);
```

Comment:
With Eclipse compiler:

    Status in line 127: 		} finally { // $line-implicitExceptionFinally.finally$ expected:<[EMPTY]> but was:<[PARTLY_COVERED]>